core/src/main/java/com/linecorp/armeria/server/DefaultRoute.java

```diff
@@ -244,7 +244,9 @@ private static String generateLoggerName(String prefix, Set<HttpMethod> methods,
                                              List<RoutingPredicate<HttpHeaders>> headerPredicates) {
         final StringJoiner name = new StringJoiner(".");
         name.add(prefix);
-        if (!methods.isEmpty()) {
+
+        // Skip if the methods is empty or it's knownMethods because it's verbose.
+        if (!methods.isEmpty() && methods != HttpMethod.knownMethods()) {
```

Comment:
(Maybe) don't we need to update `methods.isEmpty()` condition in `apply(RoutingContext routingCtx)`?
https://github.com/line/armeria/blob/3bc7bbd9363fef1a3b6d42569222ee5a4d69679d/core/src/main/java/com/linecorp/armeria/server/DefaultRoute.java#L99

Reply:
Indeed! Thanks.